src/server.rs

```diff
@@ -133,6 +134,7 @@ impl RollupBoostServer {
         boost_sync: bool,
         metrics: Option<Arc<ServerMetrics>>,
         initial_execution_mode: ExecutionMode,
+        l2_boost_factor: u64,
```

Comment:
We probably want some input validation.

Reply:
I'm thinking you can specify more than 100 if you want the builder block to have significantly higher gas used than the local block